### ISSUE TITLE
Command for dumping theme files

### ISSUE THREAD

**Issue by daveaglick** [statiqdev/Statiq#349]
Add a series of commands for:
- Listing all the files in a theme
- Dumping a specific file from the theme into the `theme` folder
- Dumping the entire theme into the `theme` folder

**Comment by daveaglick** [comment]
Which folder should files be dumped to (`input` or `theme`)? Need a way to specify.

Need to consider what happens if the file already exists. Easiest would be an overwrite prompt. Even better would be to  do some kind of diff or merge - that way this command could be used to "apply" theme changes to overrides as well. Might need to open a new issue for that feature once the easier overwrite prompt behavior is done.

**Comment by awulkan** [comment]
Would it be possible to be able to download multiple themes and have them in a themes folder? Then you could just change the config to point to the theme you currently want to use. 

If you download a theme with the same name as one that alredy exists it could just append a number or random id to it to avoid problems.

Folder structure:

/themes
-- /temeOne
-- /temeTwo
-- /temeThree
-- /temeThree1 // Added '1' to the name when downloading since "temeThree" already exists.
